src/components/existing-request/existing-request-display.vue

```diff
@@ -781,6 +782,14 @@ export default class ExistingRequestDisplay extends Mixins(
     return this.affiliateOrLogin()
   }
 
+  async amalgamateYourBusiness (): Promise<void> {
+    // safety check
+    if (!this.isNrApprovedOrConditional) return
+    await this.amalgamateNow(this.nr.entity_type_cd)
+    return this.affiliateOrLogin()
```

Comment:
I'm not sure why this is different from incorporateRegisterYourBusiness() above. It should look almost identical I think.

I think affiliateOrLogin() should end up calling amalgamateNow() -- whatever the incorporation flow does, this should do almost identically.